src/components/icons/product-icon/index.tsx

```diff
@@ -1,12 +1,24 @@
 import InlineSvg from '@hashicorp/react-inline-svg'
+import { ProductCode } from 'common/types'
 import s from './style.module.css'
 
+// TODO: is there a programatic way to build this from productNamesToIcons?
 interface ProductIconProps {
-  product: 'vault'
+  product: ProductCode
 }
 
-const productNamesToIcons = {
-  vault: require('@hashicorp/mktg-logos/product/vault/logomark/white.svg?include'),
+// TODO: what logos for hcp & sentinel?
+const productNamesToIcons: { [key in ProductCode]: any } = {
+  boundary: require('@hashicorp/mktg-logos/product/boundary/logomark/color.svg?include'),
```

Comment:
Product logos are now included in Flight icons https://github.com/hashicorp/flight/pull/341

Reply:
Oh nice!! I'll work on using these instead 😊